src/packages/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@excalidraw/utils",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
can you update the changelog and readme well ? we haven't been maintaining docs well for utils but lets update it now onwards

Reply:
I'm wondering if this should go to the `@excalidraw/excalidraw` package ?

Since I plan to use it in the vscode excalidraw extension, I wonder if it will duplicate the size of the dependencies in the node_modules folder ?